backend/src/test/java/de/jsmenues/backend/statistic/StatisticControllerTest.java

```diff
@@ -155,12 +155,18 @@ public void uploadNewDataWithResultFalse() {
     public void uploadNewDataWithResultTrue() {
         //given
         FormDataMultiPart multiPart = new FormDataMultiPart();
+        /*
+         * TODO either this parameter has to be called "chart" or its name has to be changed in 
+         * in the StatisticController.uploadNewData implementation (this might break other calls though)
+         * The test works either way, because the checking method (statisticService.updateData)
+         * is mocked to return true
+         */
         multiPart.bodyPart(new FormDataBodyPart("chartName", ""));
         multiPart.setMediaType(MediaType.MULTIPART_FORM_DATA_TYPE);
-        when(service.updateData(anyString(), any(InputStream.class), any(FormDataContentDisposition.class))).thenReturn(true);
+        when(service.updateData(nullable(String.class), nullable(InputStream.class), nullable(FormDataContentDisposition.class))).thenReturn(true);
```

Comment:
does this test do anything useful now? In fact it did not before already. The only thing this does is to ensure the service.updateData method is called, but in no way the correct take over of the parameters is verified.
do not use a wildcard in this when-command for the chartname as this is the important thing to be checked here (as you pointed out in the TODO above). Ideally for the other parameters as well, but this might be a lot harder to do